products/admin.py

```diff
@@ -0,0 +1,29 @@
+"""
+Admin settings for product app
+"""
+from django.contrib import admin
+
+# Register your models here.
+from products.models import Product
+
+
+class ProductAdmin(admin.ModelAdmin):
+    """
+    Product admin
+    """
+
+    # pylint: disable=no-member, protected-access
+    model = Product
+    all_fields = [field.name for field in Product._meta.get_fields()]
+    list_display = all_fields
+
+    list_filter = ["id", "name", "price", "quantity", "created_by"]
```

Comment:
We might not need rice and quantity filter.

Reply:
Done